code/controllers/subsystems/initialization/culture.dm

```diff
@@ -6,9 +6,24 @@ SUBSYSTEM_DEF(culture)
 	var/list/cultural_info_by_name =      list()
 	var/list/cultural_info_by_path =      list()
 	var/list/tagged_info =                list()
+	var/list/dreams = list(
+		"a familiar face", "voices from all around", "a traitor", "an ally",
+		"darkness", "light", "a catastrophe", "a loved one", "warmth", "freezing",
+		"a hat", "air", "a blue light", "blood", "healing", "power", "respect",
+		"riches", "a crash", "happiness", "pride", "a fall", "water", "flames",
+		"ice", "flying", "a voice", "the cold", "the rain", "a creature built completely of stolen flesh",
+		"a being made of light", "an old friend", "the tower", "the man with no face",
+		"an old home", "right behind you", "standing above you", "someone near by", "a place forgotten"
+	)
 
-/datum/controller/subsystem/culture/proc/get_all_entries_tagged_with(var/token)
-	return tagged_info[token]
+	var/list/credits_other =           list("ATTACK! ATTACK! ATTACK!")
+	var/list/credits_adventure_names = list("QUEST", "FORCE", "ADVENTURE")
+	var/list/credits_crew_names =      list("EVERYONE")
+	var/list/credits_holidays =        list("HOLIDAY", "VACATION")
+	var/list/credits_adjectives =      list("SEXY", "ARCANE", "POLITICALLY MOTIVATED")
+	var/list/credits_crew_outcomes =   list("PICKLED", "A VALUABLE HISTORY LESSON", "A BREAK", "HIGH", "TO LIVE", "TO RELIVE THEIR CHILDHOOD")
+	var/list/credits_topics =          list("SACRED GEOMETRY","ABSTRACT MATHEMATICS","LOVE","DRUGS","CRIME","PRODUCTIVITY","LAUNDRY")
+	var/list/credits_nouns =           list("DIGNITY", "SANITY")
```

Comment:
Dreams I can certainly see intermingled with culture, but credits seems like more of a stretch

Reply:
The sole purpose of the culture datum is to hold lore datums. It seemed as appropriate as anything given that this is all fluff. If you have a preferred subsystem I can shift it over to it.

Reply:
Renamed SSculture to SSlore.